uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/AbsenceType.tpl

```diff
@@ -1,5 +1,8 @@
-{if $action eq 1 or $action eq 2 or $action eq 8}
-  {include file="CRM/HRLeaveAndAbsences/Form/AbsenceType.tpl"}
+{if $action eq 1 or $action eq 2}
+  <div id="bootstrap-theme" class="crm-leave-and-absences-list-block"
+       data-leave-absences-leave-type-wizard>
+    <leave-type-wizard></leave-type-wizard>
+  </div>
 {else}
```

Comment:
I am not sure I get the idea, you check `if`, then render `<leave-type-wizard>`, but if `else`, you still render `<leave-type-wizard>`?

Reply:
:cut I have removed the implementation since it's no longer needed.